gitbot/lib.py

```diff
@@ -160,4 +167,4 @@ def bump_version(branch, ref_sha, author=None, url=GETSENTRY_REPO_URL, dry_run=D
     if not successful_push:
         return False, "Failed to push."
     else:
-        return True, f"Executed: {command}"
+        return True, f"Executed: {' '.join(command)}"
```

Comment:
To make the output and testing of it easier.

Reply:
shlex.join might be better

Reply:
I will look into this after lunch.